src/transport/SocketDockTransportSession.ts

```diff
@@ -0,0 +1,50 @@
+import { AgentContext, DidCommMimeType, EncryptedMessage, TransportSession } from '@credo-ts/core'
+import type { Response } from 'express'
+import { CredoError } from '@credo-ts/core'
+
+import { agentDependencies } from '@credo-ts/node'
+
+const supportedContentTypes: string[] = [DidCommMimeType.V0, DidCommMimeType.V1]
+
+export class WebSocketTransportSession implements TransportSession {
+  public id: string
+  public readonly type = 'socketdock'
```

Comment:
```suggestion
  public readonly type = 'http'
```

As transport type can be "WebSocket" or "HTTP" only

Reply:
type ws and http  transport is there for Http inbound transport and web socket inbound transport . we have added  different transport for socket dock for that i have added type as socketdock it will help us for debugging .